README.md

```diff
@@ -157,6 +157,9 @@ So just run: `geth --datadir <datadir> init ./genesis.json`
 
 ## It is recommand to run fullnode with `--tries-verify-mode none` if you want high performance and care little about state consistency
 ./geth --config ./config.toml --datadir ./node  --cache 8000 --rpc.allow-unprotected-txs --txlookuplimit 0 --tries-verify-mode none
+
+## run fullnode with Path-Base Storage Schema.
```

Comment:
```suggestion
## run fullnode with Path-Base Storage Schema
## It will enable inline state prune, keeping the latest 90000 blocks' history state by default.
```

Reply:
changed